### ISSUE TITLE
shout yield for lastshout methods and shout_home plugin fail to load

### ISSUE THREAD

**Issue by lachlan-00** [ampache/ampache#3738]
@usox I don't understand enough about how these are loaded with the generator to understand where this issue is.

```
2023-12-22T01:18:41+00:00 [user] (/var/www/music/src/Repository/Model/Shoutbox.php:66) -> Too few arguments to function Ampache\Repository\Model\Shoutbox::__construct(), 0 passed and exactly 1 expected
```

it's happening on the getTop calls which is used by the api calls and shout_home plugin
```
            $shouts = iterator_to_array(
                self::getShoutRepository()->getTop((int) $this->maxitems)
            );
```

I must have missed something in the merge/updates. i only checked the shout pages and just saw it while testing the api

**Comment by lachlan-00** [comment]
i think it's fixed with 3a7db291c, missing this

**Comment by usox** [comment]
Uh, that slipped through. Thanks.
